scripts/test.bats

```diff
@@ -47,6 +48,10 @@ contracts="GameDAO MetadataCarrier RegistryDAO TreasuryDAO TrivialDAO"
   "$exe" print-metadata > metadata.json
 }
 
+@test "Print LIGO metadata" {
+  "$ligo_exe" print-metadata > metadata.json
```

Comment:
Can you please update it now to demonstrate how to use new command line options?